packages/react-mutation-mapper/src/component/ptm/PtmAnnotationTable.tsx

```diff
@@ -46,34 +56,44 @@ export default class PtmAnnotationTable extends React.Component<
     @computed
     get columns(): Column[] {
         const pubmedRender = !_.isEmpty(this.pmidData)
-            ? (props: { original: PostTranslationalModification }) => (
-                  <PtmReferenceList
-                      pmidData={this.pmidData}
-                      pubmedIds={props.original.pubmedIds}
-                  />
-              )
-            : () =>
+            ? (props: { original: PostTranslationalModification }) =>
+                  !_.isEmpty(props.original.pubmedIds) ? (
+                      <PtmReferenceList
+                          pmidData={this.pmidData}
+                          pubmedIds={props.original.pubmedIds || []}
+                      />
+                  ) : null
+            : (props: { original: PostTranslationalModification }) =>
+                  !_.isEmpty(props.original.pubmedIds) &&
                   this.props.pubMedCache ? (
                       <i className="fa fa-spinner fa-pulse" />
                   ) : null;
```

Comment:
Right :) Actually, this is even more confusing because the ternary operation is not nested but what we set is a function that has ternary operation within. This is a workaround to trigger re-render when the pmid data updates. I'll try to make this more clear.